pywbem/_listener.py

```diff
@@ -632,6 +632,11 @@ def __init__(self, host, http_port=DEFAULT_LISTENER_PORT_HTTP,
 
             `None` means not to use a private key file. Setting up a port
             for HTTPS requires specifying a private key file.
+
+          listener_id (:term:`string`):
+            If not `None` a string that prefixes the name property of filters
+            and handler instances to help the listener identify its filters and
```

Comment:
On the term "handler": We use the term "listener destination" elsewhere, and I suggest to stay consistent. If you think it is needed, we could introduce the term "handler" somewhere, e.g. in `WBEMListener.add_server()`.

Related to consistency: I realized that in `add_server()`, the description uses the term "indication listener", which is also inconsistent and should be changed to "listener destination".